src/src/SceneNode.cpp

```diff
@@ -1,8 +1,28 @@
 #include "prefix.h"
+#include <vector>
 #include <mineola/Light.h>
 #include <mineola/Camera.h>
 #include <mineola/SceneNode.h>
 #include <mineola/Renderable.h>
+#include <glm/gtc/matrix_transform.hpp>
+
+namespace {
+float boolToSign(bool b) {
+  return b ? 1.f : -1.f;
+}
+
+// index range 0~7
+void AABBVertexes(const glm::vec3 &min, const glm::vec3 &max, std::vector<glm::vec3> &vts) {
```

Comment:
make this a member function of the AABB class, vector<vec3> AABB::Corners()
add comments to make it more readable.